libs/platform/src/lib/components/Forms/radio-group/radio-group.component.html

```diff
@@ -0,0 +1,31 @@
+<fd-form-group [isInline]="isInline">
+    <div fd-form-item *ngIf="hasNoValue">
+        <fd-radio-button
+            fd-form-control
+            name="name"
```

Comment:
Are we hard-coding "name" to the string 'name'? I think you want property binding here:
```
[name]="name"
```